### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by kanam129**

### CODE REVIEW

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 57
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 57
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 57
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 57
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 57
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 57
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 57
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 16
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5297584&bid=17340333)

**Review by NaMLiM — DISMISSED** [review]
test

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/9f7823dfc6964644bc4302465f0dad1c)](https://app.codacy.com/manual/anamnafiul99/pindad-clone?utm_source=github.com&utm_medium=referral&utm_content=NaMLiM/pindad-clone&utm_campaign=Badge_Grade_Settings)
 
 Clone of [PT. Pindad website](https://pindad.com/)
```
Comment (NaMLiM) [inline README.md:5]: test

**Review by NaMLiM — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kanam129) [reply README.md:5]: ok

**Review by NaMLiM — APPROVED** [review]

**Pull request merged**